{
    "type": "bundle",
    "id": "bundle--bf7620f0-4d7f-40a6-86f8-4e6076d3c5b6",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--fcb4adfc-7f9f-46fc-8fbf-9c32b92cfac3",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-06-11T00:24:13.061876Z",
            "modified": "2021-06-11T00:24:13.061876Z",
            "name": "CVE-2020-23313",
            "description": "There is an Assertion 'scope_stack_p > context_p->scope_stack_p' failed at js-scanner-util.c:2510 in scanner_literal_is_created in JerryScript 2.2.0",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2020-23313"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}